css/app-sidebar.scss

```diff
@@ -890,6 +886,12 @@
 	}
 }
 
+.vs__dropdown-option span {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+}
+
```

Comment:
That works quite well. I don't like that it isn't scoped but I don't see any other way as the popover is inserted directly below the body.